app_flutter/lib/build_dashboard_page.dart

```diff
@@ -68,7 +68,7 @@ class BuildDashboardPage extends StatelessWidget {
                   (String value) {
                     return DropdownMenuItem<String>(
                       value: value,
-                      child: Text(value, style: const TextStyle(color: Colors.black)),
+                      child: Text(value),
```

Comment:
Does this work in light theme? When I originally implemented this, the text was white so it blended into the dropdown.

Reply:
Light theme:
![image](https://user-images.githubusercontent.com/8620741/81090815-2177a200-8eb3-11ea-829b-987572e42a3b.png)

![image](https://user-images.githubusercontent.com/8620741/81090832-2a687380-8eb3-11ea-97a7-ba0baad203d4.png)

Reply:
How about the items in the dropdown menu?

Reply:
Ahhh I see, I'll fix that.

Reply:
Unfortunately, the best I can do right now is to use white only for dark mode and white for light mode. I'm not thrilled with that, filed https://github.com/flutter/flutter/issues/56399